### ISSUE TITLE
Text not displayed correctly in Azure instance

### ISSUE THREAD

**Issue by teocomi** [rdvojmoc/DinkToPdf#39]
Hello, the library is working fine locally when debugging with IIS, but when deployed to Azure the text is displayed as black boxes. I've tried to manually set a font face and it made no difference.
Any ideas?

![image](https://user-images.githubusercontent.com/2679513/42334889-79371acc-8076-11e8-90f8-0b5a90914984.png)

**Comment by rdvojmoc** [comment]
To me this looks like that fonts that your application is using are missing on Azure instance.

**Comment by teocomi** [comment]
Thanks for the quick reply!
I haven't explicitly set any font family for the text so it should use a default font family like times new roman or arial, no?

**Comment by rdvojmoc** [comment]
Yeah, it should, can you check if they are installed on your Azure instance?

**Comment by teocomi** [comment]
It seems that the sandbox Azure web apps run into is the problem, as it doesn't allow GDI+.
See: https://gist.github.com/tmakin/80e1905094245c857c493b7879657095#gistcomment-2068640

It's working fine from an Azure Function: https://odetocode.com/blogs/scott/archive/2018/02/14/pdf-generation-in-azure-functions-v2.aspx

Thanks again!

**Comment by mikeaxle** [comment]
@teocomi how did you deploy it azure? When I deploy an asp core 2 api from visual studio mac, the api route with this extension has a general 500 error....

**Comment by teocomi** [comment]
@mikeaxle  you should maybe debug it locally then, make sure to follow the article above that uses azure functions v2.

**Comment by mikeaxle** [comment]
@teocomi I have it working properly locally. What I am asking specifically is when you deployed it, what steps did you take? did you have to load up the 'libwkhtmltox' binaries? and where did you store them? I'll have a look at the azure functions link too.

**Comment by teocomi** [comment]
@mikeaxle I Just put libwkhtmltox.dll in the root folder of my project and clicked publish in VS, nothing else...

**Comment by honeykatrina** [comment]
Hello, for me also the library works fine locally in my .Net Core web api. After that I deploy my app to Azure App Service, F1: Free service plan. To use 32 bit version in Azure I use this response https://github.com/rdvojmoc/DinkToPdf/issues/5#issuecomment-322725241
As a result I also see black squares instead of text 
![image](https://user-images.githubusercontent.com/23028883/63770190-e4dfb980-c8d4-11e9-8977-ebaabc99963d.png)
I have tried to deploy with specified font family in the project in ObjectSettings, and without it.
Could anyone explain how to solve it? @teocomi does it work because you use Azure Function?

Update: I tried to deploy with another service plan: "Shared", and I checked which type of converter I use: SynchronizedConverter
services.AddSingleton(typeof(IConverter), new SynchronizedConverter(new PdfTools())); in my Startup.
